.github/workflows/build-packages.yml

```diff
@@ -0,0 +1,190 @@
+name: build-packages
+
+on:
+  push:
+    branches: [ 266-build-with-gitlab-actions ]
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+    
+jobs:
+
+  build-release:
+    runs-on: ubuntu-latest
+    container: rust:1.53.0
+    env:
+      TARGET: "x86_64-unknown-linux-musl"
+      BUILD_MODE: "release"
+      RUST_TC_VER: nightly-2021-06-23
+    steps:
+      - name: Checks-out repository 
+        uses: actions/checkout@v2
+      - name: Building release
+        run: |
+          apt-get update && apt-get install -y --no-install-recommends musl-tools
+          rustup install ${{ env.RUST_TC_VER }}
+          rustup default ${{ env.RUST_TC_VER }}
+          rustup target add ${{ env.TARGET }}
+          cargo build --${{ env.BUILD_MODE }} --target=${{ env.TARGET }}  
+      - name: Upload artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: bob-binaries
+          path: |
+            target/${{ env.TARGET }}/${{ env.BUILD_MODE }}/bobd
+            target/${{ env.TARGET }}/${{ env.BUILD_MODE }}/bobp
+            target/${{ env.TARGET }}/${{ env.BUILD_MODE }}/ccg
+ 
+  build-rpm:
+    runs-on: ubuntu-latest
+    needs: build-release
+    steps:
+      - name: Checks-out repository 
+        uses: actions/checkout@v2
+      - name: Download artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: bob-binaries
+      - name: Set env
+        run: echo "VERSION=$(cat bob-apps/Cargo.toml | grep ^version | head -1 | awk '{print $3}' | sed -e 's/"//g' | sed -e 's/-/./g')" >> $GITHUB_ENV
+      - name: build rpm
+        run: |
+          echo Version:${{ env.VERSION }}
+          sed -i "s/current_version/${{ env.VERSION }}/" bob.spec
+          cur_dir_name=${PWD##*/}
+          mkdir -p ./rpmbuild/{SOURCES,BUILD,RPMS,SPECS,SRPMS}
+          tar czf ./rpmbuild/SOURCES/bob-${{ env.VERSION }}.tar.gz ../${cur_dir_name}/{bobd,bobp,ccg,bob_limits.conf,bob.service} ../${cur_dir_name}/config-examples/{cluster,node,logger}.yaml --transform s/${cur_dir_name}/bob-${{ env.VERSION }}/
+          rpmbuild -ba --define "_topdir ${PWD}/rpmbuild" bob.spec
+      - name: Upload artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: bob-rpm
+          path: rpmbuild/RPMS/x86_64/bob-${{ env.VERSION }}-1.x86_64.rpm
+
+  upload-rpm:
+    runs-on: ubuntu-latest
+    needs: build-rpm
+    environment: prod
+    steps:
+      - name: Checks-out repository 
+        uses: actions/checkout@v2
+      - name: Download artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: bob-rpm
+      - name: Set env
+        run: echo "VERSION=$(cat bob-apps/Cargo.toml | grep ^version | head -1 | awk '{print $3}' | sed -e 's/"//g' | sed -e 's/-/./g')" >> $GITHUB_ENV
+      - name: Upload rpm to release
+        uses: svenstaro/upload-release-action@v2
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: bob-${{ env.VERSION }}-1.x86_64.rpm
```

Comment:
Probably you can generate and store this name one time and use it everywhere inside this workflow (try workflow level environment variable)

Reply:
Oh. I see that you parse `Cargo.toml` to generate `VERSION`. Workflow level environment variable probably won't work in this case